src/playground/sample_data_ingestion/utils/sample_data_ingestor/sample_data_ingestor.py

```diff
@@ -0,0 +1,236 @@
+from datetime import datetime, timedelta
+from shutil import copyfileobj
+from json import loads, dumps
+from os import remove, path
+from random import randint
+import gzip
+import sys
+import csv
+
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from sample_data_generator.sample_data_generator import generate_data
+from sample_data_generator.sample_data_generator import choose_field
+
+
+"""
+Function that generates random noise in the data trend (used for Anomaly Detection)
+Arguments:
+    - initial_value: The original non-anomaly value generated from generate_data_trend()
+    - anomaly_percentage: How often an anomaly occurs (minimum: 0.001)
+    - avg_min: The lowest average value in which there is no anomaly
+    - avg_max: The highest average value in which there is no anomaly
+    - abs_min: The lowest possible value, which can be an anomaly
+    - abs_max: The highest possible value, which can be an anomaly
+"""
```

Comment:
Documentation is very helpful for people to understand the code and let's follow some conventions. For example, https://google.github.io/styleguide/pyguide.html

Reply:
Done, refactored each function to have arguments, return values, and error raising.